APISpecification.yaml

```diff
@@ -171,7 +171,7 @@ paths:
           schema:
             type: string
             example: 3704ceb1-948d-11ec-a39d-005056b38ce3
-        - name: region_name
+        - name: region_id
```

Comment:
This is actually region name, according to the core data model, rather than region id. For example, for human chromosome 13, this will be "13". I am sure it has various other identifiers as well.